core/trino-main/src/main/java/io/trino/execution/DropSchemaTask.java

```diff
@@ -92,19 +92,7 @@ private static boolean isSchemaEmpty(Session session, CatalogSchemaName schema,
     {
         QualifiedTablePrefix tablePrefix = new QualifiedTablePrefix(schema.getCatalogName(), schema.getSchemaName());
 
-        // These are best efforts checks that don't provide any guarantees against concurrent DDL operations
```

Comment:
It would be beneficial to add views & materialized views in the `io.trino.testing.BaseConnectorTest#testDropNonEmptySchema` depending on:

- `SUPPORTS_CREATE_VIEW`
- `SUPPORTS_CREATE_MATERIALIZED_VIEW`

Reply:
Good idea, but we would have to test all these 3 cases separately. I don't plan to do this though. Do you want to want to follow up?

Reply:
Yes, I'll create a PR based on your changes.

Reply:
> Yes, I'll create a PR based on your changes.

many thanks!